django/db/backends/oracle/base.py

```diff
@@ -464,14 +464,22 @@ def _fix_for_params(self, query, params):
             # Handle params as dict
             args = {k: ":%s" % k for k in params.keys()}
             query = convert_unicode(query % args, self.charset)
+        elif unify_by_values:
+            # Handle params as a dict with unified query parameters by their
```

Comment:
It might be worth including an example of the transformation that's happening. As someone not familiar with any of the _fix_for_params() stuff, I can't quickly tell exactly what this is doing.